agent/virus_total_agent.py

```diff
@@ -114,11 +114,13 @@ def _prepare_targets(self, message: msg.Message) -> list[str]:
                 ip_network = ipaddress.ip_network(host)
             else:
                 version = message.data.get("version")
-                if version == 4 and int(mask) < IPV4_CIDR_LIMIT:
+                if version != 4 and version != 6:
```

Comment:
you can use `if version not in [4, 6]:`

\+ do we need this check ? 

my suggestion :

if 4 
elif 6
else ret []